src/screens/Welcome.js

```diff
@@ -43,14 +43,14 @@ class Welcome extends React.Component {
               <img className="hathor-logo" src={logo} alt="" />
               <p className="mt-4 mb-4">Welome to Hathor Wallet!</p>
             </div>
-            <p className="mb-4">This wallet is connected to a <strong>testnet</strong>.</p>
-            <p>Your HTR and other tokens may be reset at any time.</p>
-            <p>If one offers to sell some tokens to you, one is a scammer.</p>
+            <p className="mb-4">This wallet is connected to the <strong>mainnet</strong>.</p>
```

Comment:
Isn't it better to have two Welcome Screens? One for each version of the wallet? Eventually we will publish a Wallet for testnet, right?

Reply:
I'd use the same wallet and do as Trezor. If someone wants to use testnet, he should change the server address.

Reply:
Hum, seems good. The downside is that naive users may change the configuration to the testnet and be scammed.

Maybe changing the server to the testnet must have a huge alert about the risks and scammers.

Reply:
I agree. I will continue with this PR for now and create and issue to handle this